### ISSUE TITLE
Bug: Migrator.decrypt() replaces null with an empty string

### ISSUE THREAD

**Issue by Elfpkck** [luojilab/django-mirage-field#22]
If a database column has `null` values they will be replaced with the empty strings `''` after calling `Migrator.decrypt`. Found during development with `Django` and `PostgreSQL`.

**Comment by anton-kozlovsky** [comment]
Can not confirm for MySQL. `null` values are still null after running `.encrypt()` against them.

**Comment by Elfpkck** [comment]
> Can not confirm for MySQL. `null` values are still null after running `.encrypt()` against them.

Thanks for sharing the info. `.encrypt()` works as expected for me as well. But what about `.decrypt()`?

**Comment by anton-kozlovsky** [comment]
> > Can not confirm for MySQL. `null` values are still null after running `.encrypt()` against them.
> 
> Thanks for sharing the info. `.encrypt()` works as expected for me as well. But what about `.decrypt()`?

Sorry, I misread your initial message. Yes, can confirm the same behavior for MySQL as well. After running `decrypt()`, `null` values became empty strings.

**Comment by tcitry** [comment]
@Elfpkck Hi, sorry I just saw that today.

I test on my local env, and I can not confirm that.

In mirage source code, the decrypt() method is below, if `None` value will directly return `None`.

```python
    def decrypt(self, encrypted):
        if encrypted is None:
            return None
        try:
            return self.cipher.decrypt(encrypted)
        except Exception:
            return encrypted
```

In another way I searched on [Django's document](https://docs.djangoproject.com/en/3.2/ref/models/fields/#django.db.models.Field.null), and find this:

> Avoid using null on string-based fields such as CharField and TextField. If a string-based field has null=True, that means it has two possible values for “no data”: NULL, and the empty string. In most cases, it’s redundant to have two possible values for “no data;” the Django convention is to use the empty string, not NULL.

So I need more info like which field you use and how did you decrypt.

**Comment by tcitry** [comment]
@Elfpkck @anton-kozlovsky Hi, I confirm the issue with `Migrator.decrypt()`, and fixed with version `1.2.3`.

**Comment by tcitry** [comment]
@Elfpkck 
`1.2.3` has a decrypt bug, plaease switch to `1.2.4`......
